pkg/model/field.go

```diff
@@ -170,6 +175,15 @@ func NewField(
 
 	if shape != nil {
 		gte, gt, gtwp = CleanGoType(crd.sdkAPI, crd.cfg, shape, cfg)
+		if shape.Type == "structure" {
+			// "unpack" the member fields composing this struct field...
+			for _, memberName := range shape.MemberNames() {
+				cleanMemberNames := names.New(memberName)
```

Comment:
Do we also need to check for renames within the struct?

Reply:
@brycahta we *will*, yes, but since we don't *yet* support renames for nested struct fields, I will add that at a later time.